src/lib/entry-queryable.ts

```diff
@@ -15,8 +15,16 @@ export class EntryQueryable extends BaseQuery {
    * @param {string} fieldUid - field uid to select
    * @returns {EntryQueryable} - returns EntryQueryable object for chaining method calls
    */
-  only(fieldUid: string): EntryQueryable {
-    this._queryParams['only[BASE][]'] = fieldUid;
+  only(fieldUid: string|string[]): EntryQueryable {
```

Comment:
@vkalta Can you add the testcases for this implementation?

Reply:
Test cases added